### PR TITLE
Non-root user docker image.

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/issues/issues/1290

### CODE REVIEW

**Comment by schnie** [comment]
@andriisoldatenko since we're already creating an `astro` user in the base image, can we have all our derived images just switch to that user?

Maybe we even put the `USER astro` command in the base image? Would that cover all the platform images?

**Comment by andriisoldatenko** [comment]
@schnie hm... to install some deps like apk add we need to switch to USER root do some installations and than switch back to User astro.

**Comment by andriisoldatenko** [comment]
```
docker run -it astronomerinc/ap-base:3.12-1 sh
/ # id
uid=0(root) gid=0(root) groups=0(root),1(bin),2(daemon),3(sys),4(adm),6(disk),10(wheel),11(floppy),20(dialout),26(tape),27(video)
/ # whoami
root
/ #
```

**Comment by andriisoldatenko** [comment]
@schnie PTAL

**Comment by andriisoldatenko** [comment]
@adam2k looks like we need to ignore DOckerfile from codecoverage report?

**Comment by andriisoldatenko** [comment]
@schnie not sure... i see codecov error for `src/lib/deployments/logs/index.js` file.

**Comment by andriisoldatenko** [comment]
@schnie PTAL!

**Comment by andriisoldatenko** [comment]
pls don't merge found small bug in k8s cluster.

File: Dockerfile [hunk 1]
```diff
@@ -13,14 +13,23 @@
...
 
 ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 LABEL io.astronomer.docker.module="astronomer"
 LABEL io.astronomer.docker.component="houston-api"
 
+# install sudo as root
+RUN apk add --update sudo
+
+# create houston user and group
```
Comment (schnie) [inline Dockerfile:27]:
```suggestion
# Create houston user and group
```

File: Dockerfile [hunk 2]
```diff
@@ -13,14 +13,23 @@
...
-FROM astronomerinc/ap-base:3.12
+FROM astronomerinc/ap-base:3.12-1
 LABEL maintainer="Astronomer <humans@astronomer.io>"
 
 ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 LABEL io.astronomer.docker.module="astronomer"
 LABEL io.astronomer.docker.component="houston-api"
 
+# install sudo as root
```
Comment (schnie) [inline Dockerfile:24]:
```suggestion
# Install sudo as root
```

**Comment by schnie** [comment]
LGTM. Why did we switch the random function out? Just curious

**Comment by andriisoldatenko** [comment]
>LGTM. Why did we switch the random function out? Just curious
@schnie random function makes unit test flaky

File: src/lib/deployments/logs/index.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const generateLogMessage = (function* generateLogs() {
 export function generateMockLogRecords(release, component) {
   return {
     hits: {
-      hits: range(random(2)).map(() => ({
+      hits: range(5).map(() => ({
```
Comment (andriisoldatenko) [inline src/lib/deployments/logs/index.js:31]: @schnie correct usage can be:
```
hits: range(1, 2).map(() => ({
```

Comment (andriisoldatenko) [reply src/lib/deployments/logs/index.js:31]: otherwise reandom(2) can return -> 0 so we can see code coverage decreasing.

Comment (andriisoldatenko) [reply src/lib/deployments/logs/index.js:31]: i just prefer to avoid random functions in tests.

Comment (schnie) [reply src/lib/deployments/logs/index.js:31]: ah ok

**Pull request merged**